collectors/ebpf.plugin/README.md

```diff
@@ -312,6 +312,8 @@ The eBPF collector enables and runs the following eBPF programs by default:
 
 -   `fd` :  This eBPF program creates charts that show information about calls to open files.
 -   `mount`: This eBPF program creates charts that show calls for syscalls mount(2) and umount(2).
+-   `shm`: This eBPF program creates charts that show calls for syscalls
```

Comment:
I remember @vlvkobal required me to change `for` when I was writing a reference like yours :thinking: . Why did you write `calls for` instead `calls to`? (ping @DShreve2 )

Reply:
I guess I just copied it from the line above. Let's change to `to` for both.

Reply:
Good catch. We should be going with "calls to" based on what I see in the existing material. Thanks for spotting it.

Reply:
Thank you guys!